phpcs-ruleset.xml

```diff
@@ -12,6 +12,7 @@
     <!-- Minified files don't have to comply with coding standards. -->
     <exclude-pattern>*.min.css</exclude-pattern>
     <exclude-pattern>*.min.js</exclude-pattern>
+    <exclude-pattern>profiles/common/modules/features/multisite_readspeaker/js/ReadSpeaker/</exclude-pattern>
```

Comment:
Oops! These seem to be proprietary files owned by ReadSpeaker, and I'm not sure whether we are allowed to redistribute them. I will open a new ticket for this.

In scope of this ticket, this exclude pattern is excluding everything in that folder, not only the minified javascript files, please make it more specific by appending `*.js` to the end of the path.